src/test/java/brave/features/opentracing/OpenTracingAdapterTest.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.features.opentracing;
+
+import brave.Tracer;
+import brave.propagation.TraceContext;
+import io.opentracing.propagation.Format;
+import io.opentracing.propagation.TextMapExtractAdapter;
+import io.opentracing.propagation.TextMapInjectAdapter;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import zipkin.Constants;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.assertj.core.data.MapEntry.entry;
+import static zipkin.internal.Util.UTF_8;
+
+/**
+ * This shows how one might make an OpenTracing adapter for Brave, and how to navigate in and out
+ * of the core concepts.
+ */
+public class OpenTracingAdapterTest {
```

Comment:
rename this test to correspond with the type under test (ex BraveTracer etc), splitting into multiple files as needed.

Backfill tests for text map.